Is the trunk broken? Can you rebase and resubmit?

```
FAILED: ccache /opt/sdk/zephyr-sdk-0.10.3/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DBUILD_VERSION=zephyr-v2.0.0-1223-gaa3f37f90cdf -DCORE_CM4 -DHSE_VALUE=8000000 -DKERNEL -DSTM32F429xx -DUSE_FULL_LL_DRIVER -DUSE_HAL_DRIVER -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I../../../../../../kernel/include -I../../../../../../arch/arm/include -I../../../../../../include -I../../../../../../include/drivers -Izephyr/include/generated -I../../../../../../soc/arm/st_stm32/stm32f4 -I../../../../../../drivers -I../../../../../../ext/lib/crypto/tinycrypt/include -I../../../../../../ext/hal/cmsis/Include -I../../../../../../subsys/settings/include -I../../../../../../subsys/bluetooth -I/home/buildslave/src/github.com/zephyrproject-rtos/modules/hal/stm32/stm32cube/stm32f4xx/soc -I/home/buildslave/src/github.com/zephyrproject-rtos/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include -I/home/buildslave/src/github.com/zephyrproject-rtos/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include/Legacy -isystem ../../../../../../lib/libc/minimal/include -isystem /opt/sdk/zephyr-sdk-0.10.3/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/8.3.0/include -isystem /opt/sdk/zephyr-sdk-0.10.3/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/8.3.0/include-fixed -Os -imacros/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/sanity-out/nucleo_f429zi/samples/bluetooth/peripheral/sample.bluetooth.peripheral_hr.x_nucleo_idb05a1_shield/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -mthumb -mcpu=cortex-m4 -imacros/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/samples/bluetooth/peripheral=CMAKE_SOURCE_DIR -fmacro-prefix-map=/home/buildslave/src/github.com/zephyrproject-rtos/zephyr=ZEPHYR_BASE -ffunction-sections -fdata-sections -mabi=aapcs -march=armv7e-m -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/subsys/settings/src/settings_init.c.obj -MF zephyr/CMakeFiles/zephyr.dir/subsys/settings/src/settings_init.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/subsys/settings/src/settings_init.c.obj   -c /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/subsys/settings/src/settings_init.c
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/subsys/settings/src/settings_init.c: In function 'settings_backend_init':
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/subsys/settings/src/settings_init.c:143:6: error: 'DT_FLASH_ERASE_BLOCK_SIZE' undeclared (first use in this function); did you mean 'DT_FLASH_WRITE_BLOCK_SIZE'?
      DT_FLASH_ERASE_BLOCK_SIZE;
      ^~~~~~~~~~~~~~~~~~~~~~~~~
      DT_FLASH_WRITE_BLOCK_SIZE
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/subsys/settings/src/settings_init.c:143:6: note: each undeclared identifier is reported only once for each function it appears in
```

--- a/scripts/sanitycheck
+++ b/scripts/sanitycheck
@@ -581,7 +581,7 @@ class BinaryHandler(Handler):
 
         with subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=self.build_dir) as proc:
             verbose("Spawning BinaryHandler Thread for %s" % self.name)
-            t = threading.Thread(target=self._output_reader, args=(proc, harness, ))
+            t = threading.Thread(target=self._output_reader, args=(proc, harness, ), daemon=True)
             t.start()
             t.join(self.timeout)
             if t.is_alive():

